app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
```

Comment:
default behavior of form is POST, so if there is a condition where u have to write post as method dont write form by default use it.
we simply do so
form_for(resource, url: confirmation_path(resource_name))